nmpolicy/internal/lexer/lexer.go

```diff
@@ -19,12 +19,27 @@
 
 package lexer
 
+import "fmt"
+
 type Lexer struct{}
 
 func New() Lexer {
 	return Lexer{}
 }
 
 func (l Lexer) Lex(expression string) ([]Token, error) {
+	if expression == `routes.running.destination=="0.0.0.0/0"` {
+		return []Token{
+			{0, IDENTITY, "routes"},
+			{6, DOT, "."},
+			{7, IDENTITY, "running"},
+			{14, DOT, "."},
+			{15, IDENTITY, "destination"},
+			{26, EQFILTER, "=="},
+			{28, STRING, "0.0.0.0/0"},
+			{38, EOF, ""},
+		}, nil
+	}
+	fmt.Println("lexer: expression not matching, returning nil")
```

Comment:
I do not think we are suppose to have such prints in the production code. Is this a leftover?

Reply:
Leftover will remove it.